### PR TITLE
add support for klaytn

### PR DESCRIPTION
**Pull request by samuelsramko**
fix example guides
add new openapi scheme - up to date

### CODE REVIEW

**Comment by vlnevyhosteny** [comment]
Also there are some Snyk security tests that failed but I cannot take a look on it.

File: workspace.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "api-client": "packages/api-client",
     "bch": "packages/blockchain/bch",
     "bch-example": "examples/bch-example",
+    "blockchain-sdk-klaytn": "packages/blockchain/sdk-klaytn",
```
Comment (npwork) [inline workspace.json:7]: should be klaytn

File: packages/blockchain/sdk-klaytn/src/lib/sdk-klaytn.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+const blockchain = Blockchain.KLAY
+
+export const TatumKlaytnSDK = (args: SDKArguments) => {
+  const web3 = klaytnWeb3({ blockchain })
+  const api = BlockchainKlaytnService
+
+  return {
+    ...evmBasedSdk({ ...args, blockchain, web3 }),
+    api,
```
Comment (npwork) [inline packages/blockchain/sdk-klaytn/src/lib/sdk-klaytn.ts:17]: no need to expose api object. (will be removed in other packages as well)

File: examples/sdk-example/src/app/sdk.kms.example.ts [hunk 3]
```diff
@@ -6,7 +6,9 @@ const sdk = TatumSDK({ apiKey: REPLACE_ME_WITH_TATUM_API_KEY })
 
 export async function sdkKmsExample() {
   const signatureIds = await sdk.kms.getAllPending(Blockchain.BSC)
-  const tx = await sdk.kms.get(signatureIds.id)
-  await sdk.kms.complete(tx.id, tx.txId!)
+  const tx = await sdk.kms.get(signatureIds[0].id)
+  // TODO: fix this
```
[the remark this answers is not in the document]
Comment (samuelsramko) [reply examples/sdk-example/src/app/sdk.kms.example.ts:10]: no, but general SDK for KMS doesn't have abstract method sign

File: packages/shared/testing/evm-based/src/lib/evm-based.erc20.test-factory.ts [hunk 4]
```diff
@@ -143,6 +143,29 @@ export const erc20TestFactory = {
         }
       })
     },
+
+    // deploySignedTransactionChain: (sdk: SdkWithErc20Functions, testData: BlockchainTestData) => {
```
[the remark this answers is not in the document]
Comment (samuelsramko) [reply packages/shared/testing/evm-based/src/lib/evm-based.erc20.test-factory.ts:147]: removed, my internal test

**Comment by samuelsramko** [comment]
> 

New dependency - caver-js - causes this. Cant do anything with that AFAIK

**Review by vlnevyhosteny — APPROVED** [review]

**Review by vlnevyhosteny — APPROVED** [review]

**Pull request merged**